nxcomp/Loop.cpp

```diff
@@ -12942,7 +12942,7 @@ int SetLinkAdsl()
 
```

Comment:
I'd say "medium throughput networks" or (less specific) "medium speed networks" in the commit message.

Looks good otherwise. Tell me when to merge.